examples/lines_bars_and_markers/stem_plot.py

```diff
@@ -8,8 +8,22 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
+# returns 10 evenly spaced samples from 0.1 to 2*PI
 x = np.linspace(0.1, 2 * np.pi, 10)
+
+"""
```

Comment:
Thanks for the PR!  This should go in the first comment, not inside the code.

Reply:
Yes, that'd be great.  

Could you also add something like: 

```
#############################
# This example makes use of:
# * :meth:`matplotlib.axes.Axes.stem`
```

At the bottom of the example?  This will make a backlink to `.Axes.stem` that will then make a thumbnail of this example show up at:  https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.stem.html

Reply:
sure! 
will just add this at the end :+1: and update the PR